addon/styles/partials/_uni-modal.scss

```diff
@@ -38,3 +48,15 @@
     margin-right: 20px;
   }
 }
+
+.uni-modal {
```

Comment:
still testing that out since the current modal does not display the correct styling, must validate with @montoias